src/main/java/ch/njol/skript/entity/SimpleEntityData.java

```diff
@@ -279,6 +283,14 @@ public boolean equals(final @Nullable Object obj) {
 			types.add(new SimpleEntityDataInfo("wandering trader", WanderingTrader.class));
 			types.add(new SimpleEntityDataInfo("raider", Raider.class, true));
 		}
+		
+		if (Skript.isRunningMinecraft(1, 16)) {
+			types.add(new SimpleEntityDataInfo("piglin", Piglin.class));
```

Comment:
This is fine right now but in the future, piglins might be worth a separate entity data for their hunter state. So, `passive piglin` and just `piglin`, same goes for hoglins. Although `passive` might be misleading considering they would only be passive to hoglins. 

Striders also have a shivering state, so that's also worth considering.

Reply:
As for the Piglins, I considered it, but due to the changes in the API happening, I kinda wanted to wait til that settles, same goes for the upcoming Piglin Brute.

Im actually surprised the Strider has a shivering state, from what I could see it was client side.